### DISCUSSION TITLE
Add "index" to style expressions

### DISCUSSION THREAD

**Discussion by pbleyer** [maplibre/maplibre-gl-js#4016] [Ideas]
Unless there is another way to access the same functionality that I am not aware of, I see that ["index-of"](https://maplibre.org/maplibre-style-spec/expressions/#index-of) is present in the expression style spec but there is no "index" equivalent. That would allow easy access to the current element index of an array in expressions, for example to perform arithmetic operations.

**Comment by HarelM** [comment]
I would suggest to open this issue in the style-spec repo and describe how the spec operation will look and its functionality.
